### PR TITLE
Feature/43 mapping frailty scala

### PR DESCRIPTION
**Pull request by BirgitSaalfeld**
Dear @ppazos Pablo,
I finished the mapping and merged the develop branch. For some reason I don't know it runs perfectly in terminal, but fails totally in IntelliJ. 

Please review my changes. 

Thanks a lot,
Birgit

### CODE REVIEW

**Comment by BirgitSaalfeld** [comment]
It runs in IntelliJ now :) Have no clue why it changed the behaviour.

**Comment by ppazos** [comment]
@BirgitSaalfeld can you please check the conflicts with develop branch?

**Comment by BirgitSaalfeld** [comment]
@ppazos Now the branch ist up to date with the current develop branch

-

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 1]
```diff
@@ -278,15 +278,18 @@ public void createObservationUsingDefaultProfile() {
...
-            "Default profile is not supported for Observation. One of the following profiles is expected: "
-            + "[http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, "
-            + "http://hl7.org/fhir/StructureDefinition/heartrate, "
-            + "https://charite.infectioncontrol.de/fhir/core/StructureDefinition/CoronavirusNachweisTest, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/pregnancy-status, "
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Default profile is not supported for Observation. One of the following profiles is expected: " +
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:281]: @BirgitSaalfeld please fix the indentation to 4 spaces.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 2]
```diff
@@ -299,15 +302,21 @@ public void createObservationUsingUnsupportedProfile() {
...
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, "
-            + "http://hl7.org/fhir/StructureDefinition/heartrate, "
-            + "https://charite.infectioncontrol.de/fhir/core/StructureDefinition/CoronavirusNachweisTest, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/pregnancy-status, "
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+
+
+                "Profile http://hl7.org/fhir/StructureDefinition/vitalsigns is not supported for Observation. " +
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:307]: @BirgitSaalfeld please fix the indentation to 4 spaces.

**Review by ppazos — CHANGES_REQUESTED** [review]

**Review by ppazos — CHANGES_REQUESTED** [review]
@BirgitSaalfeld please check the comments about indentation.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java (hunk 2 above)
Comment (BirgitSaalfeld) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java]: done

**Comment by BirgitSaalfeld** [comment]
Dear @ppazos 
I adapted the indentation. Thank you for the detailed explanation. There is still the red circle "changed requested", though I clicked "resolved"-button after your change request. Do I have to do more?

**Review by ppazos — APPROVED** [review]

**Pull request merged**